### PR TITLE
Part1 - Restructure and expand section 5.3 (chapter5)#775

### PR DESCRIPTION
**Pull request by agrasonal**
Week2 - Milestone
Section 5.3.1: What makes a visualization effective?

### CODE REVIEW

**Comment by Vidya313** [comment]
Its a significant revision, the content is presented more meaningfully and marks proficient.

File: 04-patterns.Rmd [hunk 1]
```diff
@@ -76,16 +76,16 @@ Analytic results shouldn’t be presented to 10 decimal places when the user doe
...
 
-After answering these questions, you should able to get a better image of your ideal graph. The simple guidance for using the different types of the chart is - line charts for tracking trends over time, bar charts to compare quantities, scatter plots for a joint variation of two data items, bubble charts showing the joint variation of three data items, and pie charts to compare parts of a whole.
+After answering these questions, you should able to get a better image of your ideal graph. The simple guidance for using the different types of the chart is - line charts for tracking trends over time, bar charts to compare quantities, scatter plots for a joint variation of two data items, bubble charts showing the joint variation of three data items, and pie charts to compare parts of a whole. However, let's delve deeper into the various prsentation styles and types of common charts.
```
Comment (pragyansharma24) [inline 04-patterns.Rmd]: After answering these questions, you should be able to get a better image of your ideal graph. The simple guidance for using the different types of the chart is - line charts for tracking trends over time, bar charts to compare quantities, scatter plots for a joint variation of two data items, bubble charts showing the joint variation of three data items, and pie charts to compare parts of a whole. However, let's delve deeper into the various presentation styles and types of common charts.

Comment (pragyansharma24) [reply 04-patterns.Rmd]: @agrasonal : in line 88 
1. be word is missing from "you should be able to"
2. spelling of the word presentation

**Comment by pragyansharma24** [comment]
Please check my comments on line 88.

File: 04-patterns.Rmd (hunk 1 above)
Comment (agrasonal) [reply 04-patterns.Rmd]: Thanks @pragyansharma24 for the change suggestions. I have updated the file as per your suggestions.

**Pull request merged**